dist/styles/main.css

```diff
@@ -67,9 +79,14 @@ body.darwin .menu .btn-group {
     top: 0;
     width: 100%;
     height: 100%;
-    background-color: var(--neutralLight);
+    background-color: #0008;
     overflow: hidden;
 }
+@media (prefers-color-scheme: light) {
+    .settings-container {
+        background-color: #0004
+    }
```

Comment:
Why is the background for settings changed?

Reply:
I tried to make settings page acrylic, but failed. I'm reverting it.

Reply:
Commit https://github.com/yang991178/fluent-reader/pull/72/commits/8e8823343744cfd1b8e282dce5012bba81fa8de2 reverted it.